docs/automations/set-up-natural-language-search.md

```diff
@@ -156,6 +156,20 @@ by natural language or similar images over the corresponding image data.
 
 In this tutorial, we learned how to extract and upload vector embeddings over your image data automatically and manually.
 
+## Examples
```

Comment:
Isn't this covered earlier in this page? Wondering what additional benefit this change brings

<img width="646" alt="image" src="https://github.com/user-attachments/assets/8f836e45-d34f-48a2-b1d0-5e434ce429ad">

Reply:
Good callout, it seems clients miss the example section https://kolena-io.slack.com/archives/C07E6AFUNQ5/p1729690899248259

Also, I will add a section for what dataFrame is require and change the name from "Example" to "Note".

Reply:
Right, I suspect this might have been in part because they were not specifically looking for `Setting up Natural Language Search` so did not read that doc

Reply:
maybe better to create a separate doc for uploading embeddings? I will update this

Reply:
Yea I feel like that'd be helpful, that that doesn't necessarily have anything to do natural language search either but mainly for embedding visualization

Reply:
Updated PR based on the feedback